integration-tests/jaxrs/app/pom.xml

```diff
@@ -46,10 +46,23 @@
             <artifactId>shiro-jaxrs</artifactId>
         </dependency>
 
+        <!-- needed by javafaker, override -->
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>2.0</version>
+        </dependency>
+
         <dependency>
             <groupId>com.github.javafaker</groupId>
             <artifactId>javafaker</artifactId>
             <version>1.0.2</version>
+            <exclusions>
```

Comment:
Are you sure this exclusion is necessary?
Wouldn't the inclusion of latest snakeyaml reflect here? I would think so

Reply:
Sadly yes, because v2 of snakeyaml doesn't have an android classifier, so it cannot be overridden anymore. v1.x is heavily outdated with existing CVEs.